app/jobs/concerns/import_job.rb

```diff
@@ -63,42 +53,15 @@ def category_model
     @category_model ||= category.constantize
   end
 
-  def stats
-    @stats ||= Hash.new(0)
-    @stats[:shl_case_numbers] = Hash.new(0) unless @stats.key?(:shl_case_numbers)
-    @stats
-  end
-
-  def increment_stats(attrs, persisted)
-    stats[:row_count] += 1
-    if persisted
-      stats[:rows_imported] += 1
-      stats[:shl_case_numbers][attrs['shl_case_number']] += 1 if attrs.key?('shl_case_number')
-    else
-      stats[:rows_not_imported] += 1
-    end
-  end
-
-  # We're receiving date values in a format different than what ActiveRecord expects, we need a mechanism
-  # for applying transformations to values before they get to the model.
-  def preprocess_attribute_values(attrs)
-    attrs
-  end
-
-  def translate_attribute_names(attrs)
-    attrs
-  end
```

Comment:
Translating and pre-processing logic moved to a new `create_from_csv_data` method on the importable models.